gc/verbose/VerboseHandlerOutput.cpp

```diff
@@ -482,6 +502,14 @@ MM_VerboseHandlerOutput::handleCycleEnd(J9HookInterface** hook, uintptr_t eventN
 	} else {
 		writer->formatAndOutput(env, 0, "<cycle-end %s />", tagTemplate);
 	}
+
+
+	if ((event->cycleType == OMR_GC_CYCLE_TYPE_GLOBAL) && (event->fixHeapForWalkReason != FIXUP_NONE)) {
```

Comment:
we might have a fixup from a local GC (in a rare case, contraction of Nursery), after all
regardless, I'd keep it simple, and keep only the latter condition
reverse the order of arguments in the operator (constant first)

Reply:
Ok no problem, sounds good.